src/components/ChatEntry.js

```diff
@@ -1,22 +1,49 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
 const ChatEntry = (props) => {
+  let side =
+    props.sender === 'Vladimir' ? 'chat-entry local' : 'chat-entry remote';
+
+  const onLikeButtonClick = () => {
+    let updatedMessage = {
+      id: props.id,
+      sender: props.sender,
+      body: props.body,
+      timeStamp: props.timeStamp,
+      liked: !props.isLiked,
+    };
+
+    props.onUpdate(updatedMessage);
+  };
```

Comment:
hmm you are giving this function a lot of editing power when it only needs to update the `liked` property. Think about giving the function access only to `liked`.